R/run_mcmc.R

```diff
@@ -110,14 +110,26 @@ run_mcmc_p <- function(dt, priorObj, n.chains, n.adapt, n.burn, n.iter, seed, pa
     cat("seed_list:", seed_list, "number of dataset", length(dt), "\n")
   )
 
-  message(n.cores, " clusters are being used")
+  ps_message(n.cores, " clusters are being used")
   cl <- parallel::makeCluster(n.cores)
   doParallel::registerDoParallel(cl)
 
+  # Load psborrow on the clusters
+  local.psborrow.quiet <- getOption('psborrow.quiet') # To pass to clusters
+  parallel::clusterExport(cl, varlist =  c('local.psborrow.quiet'), envir = environment())
+  parallel::clusterEvalQ(cl, {
+    try( # For development purposes, we will have to call `devtools::load_all()`
+      if(grep('psborrow',rstudioapi::getActiveProject())) {
+        devtools::load_all()
```

Comment:
I would replace this with the underlying package i.e. `pkgload::load_all()` additionally you will need to add pkgload then to the suggests field of the DESCRIPTION (if it isn't already there)